cookbooks/munin/recipes/server.rb

```diff
@@ -95,6 +101,21 @@
   mode 0o755
 end
 
+# simple shell script to dump RRD data to a file
+cookbook_file "/usr/local/bin/rrddump" do
+  source "rrddump.sh"
+  owner "root"
+  group "root"
+  mode 0o755
+end
+
+template "/etc/cron.d/rrddump" do
+  source "rrddump.cron.erb"
+  owner "root"
+  group "root"
+  mode 0o755
+end
```

Comment:
We need something to expire old dumps or we will eventually run out of disk space.